src/serialization/toSerializableFormat.js

```diff
@@ -32,15 +32,18 @@ function saveStatus(originalState, serializableState) {
  * @returns {*}
  */
 function transformSubstate(originalSubState) {
-  if (_.isPlainObject(originalSubState)) {
-    // eslint-disable-next-line  no-use-before-define
-    return toSerializableFormat(originalSubState);
-  } else if (_.isArray(originalSubState) && originalSubState[STATUS]) {
+  if (_.isArray(originalSubState)) {
+    if (!originalSubState[STATUS]) {
```

Comment:
Agree, will test and fix. I used `isObjectLike` so maybe that is the reason why I left it.  Thanks.